tests/test_corpus.py

```diff
@@ -25,6 +25,7 @@
     thai_syllables,
     thai_synonym,
     thai_words,
+    wikipedia_titles,
```

Comment:
Do you mind to put it in alphabetical order please?

Reply:
Oh sorry, fixed!